src/MagentoHackathon/Composer/Magento/Parser/PackageXmlParser.php

```diff
@@ -64,6 +64,8 @@ protected function parseMappings()
                     foreach ($target->children() as $child) {
                         foreach ($this->getElementPaths($child) as $elementPath) {
                             $relativePath = $basePath . '/' . $elementPath;
+                            //remove the any trailing './' or '.' from the targets base-path.
+                            $relativePath = preg_replace('#^[\/\.]*#', '', $relativePath);
```

Comment:
They are not exactly the same, what do you suggest without introducing another dependency? Maybe we can just wrap the ones in `GitIgnore` into a private method?